docs/DeveloperGuide.md

```diff
@@ -154,6 +154,42 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Add feature
+
+The `add` mechanism is facilitated by `AddressBook`. The implementation of adding a `Candidate` through the `add` command has been enhanced on the existing approach. It extends `Command`. The input parameters of the `add` command has been tweaked slightly, introducing a few more related entities.
+
+List of new entities:
+1. `StudentId`
+2. `Course`
+3. `Seniority`
+4. `ApplicationStatus`
+5. `InterviewStatus`
```

Comment:
Maybe you can add in your notes that `ApplicationStatus` and `InterviewStatus` will be set to default. And then maybe in the `EditCommand` I could elaborate further how to edit these attributes

Reply:
Will do @tiewweijian!